examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -143,6 +143,15 @@ model:
     num_classes: -1
     vocabulary: []
 
+  # config for InterCTC loss: https://arxiv.org/abs/2102.03216
```

Comment:
Are we going to support stochastic depth for Transducer models?

Reply:
It's supported, but only for ConformerEncoder. The other encoders are possible, but need to check the details, so let's do it in another PR

Reply:
Why not adding stochastic_depth_start_layer and their params to the encoder section?
You have just added the InterCTC section, right?
Is it OK to just support Conformer. I meant for Conformer-Transducer configs.

Reply:
added to the example configs (if that's what you meant)